### PR TITLE
Update repositories.txt

### PR DESCRIPTION
**Pull request by Silver-Fang**
Update URL of my repo which was changed to a more appropriate name

### CODE REVIEW

**Review by per1234 — COMMENTED** [review]
Hello @Silver-Fang. I am confirming your request.

Unlike submissions, this type of request is still handled manually. The Library Manager index maintainer occasionally comes through to clear all the outstanding requests. They will merge this pull request when it is done. You will get a notification from GitHub at that time.

After that, you can monitor the indexing of the library via its dedicated logs web page.
Information about that here:
https://github.com/arduino/library-registry/blob/main/FAQ.md#can-i-check-on-library-releases-being-added-to-library-manager

**Review by MatteoPologruto — APPROVED** [review]
Done:
```
2024/09/12 16:06:03 Loaded 7292 libraries from DB
Changing URL of library Cpp_Standard_Library from https://github.com/Silver-Fang/ArduinoSTL.git to https://github.com/Silver-Fang/Cpp_Standard_Library.git
Success!
```

**Pull request merged**
